src/functions-and-arrays.js

```diff
@@ -1,19 +1,43 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+  if (num1<num2) {
+    return num2;
+  }
+  else if (num1>num2) {
+    return num1;
+  }
+  else if (num1=num2) {
+    return num1 || num2;
```

Comment:
- always keep your code as short as possible, meaning leave only neccesary code :) Soo, here you might as well write `return num1;`
- when you are doing comparison, always use `==` or `===`. Because with single` =`  you are asigning a new value to a variable